lib/miq_automation_engine/engine/miq_ae_service.rb

```diff
@@ -147,7 +147,7 @@ def instantiate(uri)
     def drb_return(obj)
       # Save a reference to the object, so that we control when it gets deleted.  Otherwise, Ruby Garbage Collection may remove it prematurely.
       # If it is removed prematurely and then referenced by the method, we get a DRb recycled object error
-      @drb_server_references << obj
+      #@drb_server_references << obj
```

Comment:
Since this change is hard to validate we were planning to leave `drb_return` as a no-op for now and remove it in a later PR once this change gets a little burn-in time.  Longer term we want to interact with the Rest API instead of Drb and this would be removed then, if not before.